worldedit-core/src/main/java/com/sk89q/worldedit/internal/util/ErrorReporting.java

```diff
@@ -30,12 +30,17 @@
  */
 public class ErrorReporting {
     public static void trigger(Actor actor, Throwable error) {
-        actor.printError(TranslatableComponent.of("worldedit.command.error.report"));
-        actor.print(
-            TextComponent.builder(error.getClass().getName() + ": " + error.getMessage())
-                .hoverEvent(HoverEvent.showText(TextComponent.of(Throwables.getStackTraceAsString(error))))
-                .build()
-        );
+        boolean showDetailedError = actor.hasPermission("worldedit.error.detailed");
+
+        actor.printError(TranslatableComponent.of("worldedit.command.error"));
+        if (showDetailedError) {
+            actor.printError(TranslatableComponent.of("worldedit.command.error.report"));
```

Comment:
I would rather not add a new message:
```suggestion
        actor.printError(TranslatableComponent.of("worldedit.command.error.report"));
        if (showDetailedError) {
```